### ISSUE TITLE
Add optional routing via react-router-redux

### ISSUE THREAD

**Issue by janek26** [stylesuxx/generator-react-webpack-redux#31]
Routing option would be fine.
Generators for new routes are also desired.

**Comment by stylesuxx** [comment]
Yes, I am working on such a feature. I am still evaluating options, but it will be implemented in the near future ;-)

**Comment by guddas** [comment]
for a start take a look at https://github.com/guddas/generator-react-webpack-redux/tree/route
